app.js

```diff
@@ -25,20 +32,77 @@ app.get('/', (req, res) => {
 //  POST  /recipes route
 
 
+app.post('/recipes', (req,res) => {
+
+    Recipe.create(req.body)
```

Comment:
Something that we can  do in order to  keep the code easy to read is destructuring the properties from it 
```suggestion
const { title, instructions, level, ingredients, image ,duration, isArchived, created} = req.body
  Recipe.create({
    title,
    instructions,
    level,
    ingredients,
    image,
    duration,
    isArchived,
    created,
  })
```